docs/worklow-configuration-how-to.md

```diff
@@ -55,3 +82,78 @@ DiscoverGranules:
         buckets:
           internal: '{$.meta.collection.name}-bucket'
 ```
+## How to specify a file location in a bucket
+
+Granule files can be placed in folders and subfolders in buckets for better organization. This is done by setting a `url_path` in the base level of a collection configuration to be applied to all files. To only affect placement of a single file, the `url_path` variable can be placed in that specific file of the collection configuration. There are a number of different ways to populate `url_path`.
+
+### Hardcoding file placement
+
+A file path can be added as the `url_path` in the collection configuration to specify the final location of the files. For example, take the following collection configuration
+
+```
+{
+  "name": "MOD09GQ",
+  "version": "006",
+  "url_path": "example-path",
+  "files": {
+    {
+      "bucket": "protected",
+      "regex": "^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006.[\\d]{13}\\.hdf$",
+      "sampleFileName": "MOD09GQ.A2017025.h21v00.006.2017034065104.hdf",
+      "url_path": "file-example-path"
+    },
+    {
+      "bucket": "private",
+      "regex": "^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006.[\\d]{13}\\.hdf\\.met$",
+      "sampleFileName": "MOD09GQ.A2017025.h21v00.006.2017034065104.hdf.met"
+    }
+  }
+}
+```
+
+The first file, `MOD09GQ.A2017025.h21v00.006.2017034065104.hdf` has its own `url_path` so the resulting file path might look like `s3://cumulus-test-sandbox-protected/file-example-path/MOD09GQ.A2017025.h21v00.006.2017034065104.hdf`.
+The second file, `MOD09GQ.A2017025.h21v00.006.2017034065104.hdf.met`, does not have it's own `url_path` so it will use the collection `url_path` and have a final file path of `s3://cumulus-test-sandbox-protected/example-path/MOD09GQ.A2017025.h21v00.006.2017034065104.hdf.met`.
+
+### Using a template for file placement
+
+Instead of hardcoding the placement, the `url_path` can be a template to be populated with metadata during the move-granules step. For example:
```

Comment:
We need an explanation here that the Move Granules step needs to be added to the workflow in order for this to work.